lib/tasks/test_security.rake

```diff
@@ -9,8 +9,14 @@ namespace :test do
       require "vmdb/plugins"
       require "brakeman"
 
+      # Brakeman 5.0+ excludes files in the vendor directory even if they're included
+      # via engine paths.  We patch it below so that requested engine paths are not excluded.
+      require 'brakeman/app_tree'
+      require Rails.root.join('lib/extensions/brakeman_excludes_patch')
+      Brakeman::AppTree.prepend(BrakemanExcludesPatch)
+
       app_path = Rails.root.to_s
-      engine_paths = Vmdb::Plugins.paths.values
+      engine_paths = Vmdb::Plugins.paths.except(ManageIQ::Schema::Engine).values
```

Comment:
Can this be moved out into a separate commit?  This way we can revert the other commit when upstream changes.  This technically is only needed because these changes surfaced false positives, but handling the false positives is a separate thing.

Reply:
Done. I had it separate but squashed it.  haha.  I agree, it should be separate.